pkg/trace/api/otlp.go

```diff
@@ -208,12 +340,12 @@ func (o *OTLPReceiver) ReceiveResourceSpans(ctx context.Context, rspans ptrace.R
 		hostFromMap(rattr, "_dd.hostname")
 	}
 	// TODO(songy23): use AttributeDeploymentEnvironmentName once collector version upgrade is unblocked
-	_, env := getFirstFromMap(rattr, "deployment.environment.name", semconv.AttributeDeploymentEnvironment)
+	_, env := transform.GetFirstFromMap(rattr, "deployment.environment.name", semconv.AttributeDeploymentEnvironment)
```

Comment:
This change is just moving the "GetFirstFromMap" function to a new module so that it's accessible by the other functions in `transform`; there shouldn't be any functional change to v1.